docs/index.md

```diff
@@ -4,14 +4,14 @@ Go to the livestream: [stream.hoobs.live](https://stream.hoobs.live/).
 
 
 ## Youtube Live Stream
-* [Chat Commands](../youtube#commands)
-* [Blex Bot](../youtube#blexbot)
+* [Chat Commands](../docs/youtube/index.md)
```

Comment:
we need to change the ../ to just youtube and index.md is assumed

Reply:
Didn't work when I tested it on GitHub.